enterprise/frontend/src/embedding-sdk/test/__support__/ui.tsx

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line no-restricted-imports
```

Comment:
we seem to have exported this type. Can't we use that?
https://github.com/metabase/metabase/blob/7da3631d787c1eb393512c34909864f94b479208/enterprise/frontend/src/embedding-sdk/lib/theme/get-embedding-theme.ts#L17